File: 0344-reverse-string/0344-reverse-string.py
class Solution:
    def reverseString(self, s: List[str]) -> None:
        """
        Do not return anything, modify s in-place instead.
        """
        
        start = 0
        end = len(s) - 1

        while start < end:
            x = s[start]
            s[start] = s[end]
            s[end] = x

            start += 1
            end -= 1

File: 0344-reverse-string/0344-reverse-string.java
class Solution {
    public void reverseString(char[] s) {
        if(s.length == 1) {
            return;
        }
         
        for(int i = 0; i < s.length/2; i++) {
            char ch = s[i];
            s[i] = s[s.length - i - 1];
            s[s.length - i - 1] = ch;
        }
    }
}